indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/PartialDimensionDistributionTask.java

```diff
@@ -106,7 +106,8 @@
     );
   }
 
-  @VisibleForTesting  // Only for testing
+  @VisibleForTesting
+    // Only for testing
```

Comment:
Ok... that comment has been there for a long time... the only reason it shows as my change is because of automatic formatting from Intelli-j. I will remove the message in next commit.